### DISCUSSION TITLE
Old MDI still alive?

### DISCUSSION THREAD

**Discussion by terminator356** [muse-sequencer/muse#1185] [Team Posts]
Lookin' sharp @kybos ! You're on a roll !

I opened a project from last year and it looks like this,
 I can resize and move the Arranger window. What do you make of this?

![Arranger_mdi_with_old_project_1](https://user-images.githubusercontent.com/5422362/97512633-5cd5c380-1960-11eb-9f1f-58af3a403cb5.png)

**Comment by terminator356** [comment]
I clicked Windows > Arranger and it cured it, Arranger went full size again.

**Comment by kybos** [comment]
I saw this problem before and thought I had fixed it. Obviously there is still a gap somewhere...
The reason is the arranger window geometry that is stored in the old file. It overrides the maximized subwindow. This is quite an unfortunate weakness of the Qt tabbed MDI mode, it doesn't maximize the tabbed windows automatically, it has to be done "manually" in all relevant places.
I will check. Thanks for the report.

**Comment by kybos** [comment]
I made some changes, please try if it helped. I couldn't reproduced it with my older files so I don't know.
If not, please send me the file (you can delete the music of course, the skeleton with the settings should do).

**Comment by terminator356** [comment]
Re-tested: Seems OK now, Arranger opened full size upon loading.
Thanks!

**Comment by terminator356** [comment]
Here's a link to the song file in case you want to see what might have caused it.
Any wave files will be missing of course.
[https://www.dropbox.com/s/ognwa9cwc11susc/latency_drum_machine_test_1.med?dl=0](url)

[ Edit ] The link doesn't seem to work here. Copy and paste it into a browser and it should work.
